src/Twilio/Security/RequestValidator.cs

```diff
@@ -124,22 +125,33 @@ private static bool SecureCompare(string a, string b)
             return mismatch == 0;
         }
 
-        private string RemovePort(UriBuilder uri)
+        private string PreserveHostnameCasing(string url)
```

Comment:
Should move this to below `Add/RemovePort` so the code reads from top-to-bottom.